coral/coral/media/js/views/components/reports/activity.js

```diff
@@ -10,10 +12,13 @@ define([
     'views/components/reports/scenes/json',
     'views/components/reports/scenes/classifications',
     'views/components/reports/scenes/location',
-    'views/components/reports/scenes/protection'
-], function($, _, ko, arches, resourceUtils, reportUtils) {
+    'views/components/reports/scenes/protection',
+    'views/components/reports/scenes/default',
+    // 'default-report'
+], function(activityTemplate, $, _, ko, arches, resourceUtils, reportUtils) {
     return ko.components.register('activity-report', {
         viewModel: function(params) {
+            console.log(activityTemplate)
```

Comment:
This can go btw, but if it's next PR that's fine (as it's taking a long time to run through)

Reply:
alright I'll remove it